.github/workflows/push-deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy to WordPress.org
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install Composer Dependencies
```

Comment:
Before instlaling composer, I could add this:

```
- name: Validate composer.json and composer.lock
   run: composer validate --strict
```

Reply:
I've added this.